components/simulator/src/components/chatWindow.tsx

```
@@ -30,7 +30,11 @@ const ChatMessage: React.FC<{ message: Message }> = ({ message }) => {
   );
 };
 
-const ChatWindow: React.FC<{ ai: InteractWithAI }> = ({ ai }) => {
+const ChatWindow: React.FC<{ ai: InteractWithAI | null, onError: Function }> = ({ ai, onError }) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)